airbyte-webapp/src/components/ContentCard/ContentCard.tsx

```diff
@@ -1,59 +1,26 @@
 import React from "react";
 import styled from "styled-components";
 
-import { H5 } from "components";
+import { H5, Card } from "components";
 
 type IProps = {
   title?: string | React.ReactNode;
-  description?: string | React.ReactNode;
-  attentionMessage?: string | React.ReactNode;
   className?: string;
   onClick?: () => void;
 };
 
-const Card = styled.div`
-  background: ${({ theme }) => theme.whiteColor};
-  border-radius: 8px;
-  box-shadow: 0 1px 2px ${({ theme }) => theme.shadowColor};
-  border: 1px solid ${({ theme }) => theme.greyColor20};
-`;
-
 const Title = styled(H5)`
   padding: 25px 25px 22px;
-  border-bottom: 1px solid ${({ theme }) => theme.greyColor20};
+  //border-bottom: 1px solid ${({ theme }) => theme.greyColor20};
```

Comment:
@Jamakase should this be removed?

Reply:
Actually no, this is part of new UI where we update borders and shadows. Decided to leave it as a comment to check all places more properly.